src/events/EventsSection.tsx

```diff
@@ -122,60 +230,269 @@ export const EventsSection = () => {
+                    >
+                      <Select
+                        id="kc-eventType"
+                        name="eventType"
+                        data-testid="event-type-searchField"
+                        chipGroupProps={{
+                          numChips: 1,
+                          expandedText: "Hide",
+                          collapsedText: "Show ${remaining}",
+                        }}
+                        variant={SelectVariant.typeaheadMulti}
+                        typeAheadAriaLabel="Select"
+                        onToggle={onSelectToggle}
+                        selections={selectedEvents}
+                        onSelect={(_, value) => {
+                          const option = value.toString();
+                          register("eventTypes");
+                          if (selectedEvents?.includes(option)) {
+                            setSelectedEvents(
+                              selectedEvents.filter((item) => item !== option)
+                            );
+                            setValue(
+                              "eventTypes",
+                              selectedEvents.filter((item) => item !== option)
+                            );
+                            setIsDirty(true);
+                          } else {
+                            setSelectedEvents([...selectedEvents, option]);
+                            setValue("eventTypes", [...selectedEvents, option]);
+                            setIsDirty(true);
+                          }
+                        }}
+                        onClear={clearSelection}
+                        isOpen={selectOpen}
+                        aria-labelledby={"eventType"}
+                        chipGroupComponent={chipGroupComponent()}
+                      >
+                        {events?.enabledEventTypes?.map((option, index) => (
+                          <SelectOption
+                            key={`eventType-${index}`}
+                            value={option}
+                          />
+                        ))}
+                      </Select>
+                    </FormGroup>
+                    <FormGroup
+                      label={t("client")}
+                      fieldId="kc-client"
+                      className="keycloak__user_events_search__form_label"
+                    >
+                      <TextInput
+                        ref={register()}
+                        type="text"
+                        id="kc-client"
+                        name="client"
+                        data-testid="client-searchField"
+                        onChange={(_val, event) => {
+                          const { value } = event.currentTarget;
+                          setValue("client", value);
+                          setIsDirty(true);
+                        }}
+                        defaultValue={
+                          formValues?.client ? formValues?.client : ""
```

Comment:
This can be shortened.
```suggestion
                          formValues?.client ??  ""
```

Reply:
fixed, thanks